dotnet3.5/S3/CopyObjectExample/CopyObject/CopyObject.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CopyObject]
```

Comment:
Add a blank line after the license declaration -- here and throughout.

Reply:
Done in all projects.